src/language/en-US.json

```diff
@@ -772,10 +772,14 @@
   "login_account": "Login",
   "login": "Login",
   "create_a_new_account": "Create a new account",
+  "go_to_login": "Go to login",
   "Access_My_Account": "Access My Account",
   "Do_not_have_Account": "You don't have an account yet?",
+  "Existing_Account": "Existing account",
```

Comment:
`Do you have an account already?`

Reply:
![image](https://user-images.githubusercontent.com/18167466/76855632-7b34f600-688c-11ea-853c-00dd7f485de8.png)
like this?

Reply:
1. remove the comma `,` after the question sign.

2. Replace `Go to login` with `Click here to login now.`

Reply:
O(∩_∩)O